editor.planx.uk/src/pages/Team/index.tsx

```diff
@@ -68,13 +94,31 @@ const Team: React.FC = () => {
 
   const route = useCurrentRoute();
 
+  const [showCardGrid, toggleCardGrid] = useStore((state) => [
+    state.showCardGrid,
+    state.toggleCardGrid,
+  ]);
+
+  const viewType = showCardGrid ? "grid" : "row";
+
+  const handleViewChange = (
+    _event: React.MouseEvent<HTMLElement>,
+    newView: "grid" | "row",
+  ) => {
+    if (!newView) return;
+
+    const isGrid = newView === "grid";
+    if (isGrid !== showCardGrid) {
+      toggleCardGrid();
+    }
+  };
```

Comment:
I think we can skip this and just call `() => toggleCardGrid()` from the onClick of the component.